tests/lax_test.py

```diff
@@ -4485,15 +4485,16 @@ def regions_with_inaccuracies_keep(*to_keep):
     elif name == 'tanh':
       regions_with_inaccuracies_keep('ninf', 'pinf', 'ninfj', 'pinfj')
 
-    elif name == 'arccos':
+    elif name == 'arccos' and jax._src.lib.version <= (0, 4, 33):
```

Comment:
You can remove the version guard, the minimum is now 0.4.38.

Reply:
Removed, thanks!